src/frontend/screens/Library/components/GameCard/index.tsx

```diff
@@ -165,7 +165,10 @@ const GameCard = ({
 
   const isHiddenGame = libraryState.isGameHidden(appName)
   const isBrowserGame =
-    installPlatform === 'Browser' || installPlatform === 'web'
+    installPlatform === 'Browser' ||
+    installPlatform === 'web' ||
+    // @ts-expect-error will need to update valist SDK first
+    installPlatform === 'webgl'
```

Comment:
webgl is not a browser game in this context. it is a download url to a webgl build of a game. it should not be surfaced in the client

Reply:
fixed